src/network/connectionpool.py

```diff
@@ -253,8 +259,8 @@ def loop(self):
                     self.startListening()
                 else:
                     for bind in re.sub(
-                        "[^\w.]+", " ",
-                        BMConfigParser().safeGet('network', 'bind')
+                            "[^\w.]+", " ",     # pylint: disable=anomalous-backslash-in-string
```

Comment:
maybe this should be a `'` instead of `"`?

Reply:
Okay, will update the code.